keystone_api/apps/allocations/factories.py

```diff
@@ -0,0 +1,126 @@
+"""Factories for the allocations app, used for generating test data.
```

Comment:
The other project modules use a same standardized docstring across all modules of the same type. E.g., all `models.py` modules have the identical module level docstring describing what models are used for. 

For factories, how about something like:

```python
"""Factories for creating mock database records.

Factory classes are used to generate realistic mock data for use in
testing and development. Each class encapsulates logic for constructing
a specific model instance with sensible default values. This streamlines
the creation of mock data, avoiding the need for hardcoded or repetitive
setup logic.
"""
```